x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_results.tsx

```diff
@@ -380,10 +412,23 @@ export const LogRateAnalysisResults: FC<LogRateAnalysisResultsProps> = ({
           <FieldFilterPopover
             disabled={!groupResults || isRunning}
             disabledApplyButton={isRunning}
-            uniqueFieldNames={uniqueFieldNames}
+            disabledApplyTooltipContent={disabledFieldFilterApplyButtonTooltipContent}
+            helpText={fieldFilterHelpText}
+            uniqueItemNames={uniqueFieldNames}
             onChange={onFieldsFilterChange}
           />
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <ColumnFilterPopover
+            disabled={isRunning}
+            disabledApplyButton={isRunning}
+            disabledApplyTooltipContent={disabledColumnFilterApplyButtonTooltipContent}
+            helpText={columnsFilterHelpText}
+            itemType="columns"
+            uniqueItemNames={(groupResults ? commonColumns : significantItemColumns) as string[]}
```

Comment:
`commonColumns` and `significantItemColumns` are being as the list of items to show, but they are not internationalized

Reply:
Internationalized in 72f120e9a39c579999d4b3aabcf548ff88d6fdf0